recipes-devtools/nsight-systems/nsight-systems-cli_2020.5.3.17-1.bb

```diff
@@ -0,0 +1,53 @@
+SUMMARY = "NVIDIA Nsight Systems is a multi-core CPU sampling profiler that \
+    provides an interactive view of captured profiling data, helping improve \
+    overall application performance."
+HOMEPAGE = "https://developer.nvidia.com/nsight-systems"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = " \
+    file://opt/nvidia/nsight-systems-cli/2020.5.3/NVIDIA_SLA.pdf;md5=f7646eea4f29c4f708ea0aa5bc6778a6 \
```

Comment:
Is there no plain text copy of the license anywhere in the package?  Plain text would be better.

Reply:
No

Reply:
Yeah, I see.  Oh, well.